crates/consensus/src/lib.rs

```diff
@@ -63,6 +63,7 @@ pub mod crypto;
 pub mod error;
 
 pub mod extended;
```

Comment:
This module should probably be private to disambiguate. It's contents are publicly re-exported now.